resources/js/components/Litter/Tags.vue

```diff
@@ -1,21 +1,21 @@
 <template>
-	<div>
+    <div>
         <ul class="container">
-            <li v-for="category in categories" class='admin-item'>
-
+            <li v-for="category in categories" class="admin-item">
                 <!-- Translated Category Title -->
                 <span class="category">{{ getCategory(category.category) }}</span>
 
                 <!-- List of tags in each category -->
                 <span
                     v-for="tags in Object.entries(category.tags)"
-                    v-html="getTags(tags, category.category)"
-                    class="tag is-large is-info litter-tag"
-                    @click="removeTag(category.category, tags[0])"
```

Comment:
We need to keep @click="removeTag" here. Also, bring v-html back please. I prefer the readability